transformation/plugins/org.jboss.tools.fuse.transformation.editor/src/main/java/org/jboss/tools/fuse/transformation/editor/internal/util/Util.java

```diff
@@ -64,41 +67,35 @@
 import org.jboss.tools.fuse.transformation.editor.internal.dozer.DozerResourceClasspathSelectionDialog;
 import org.jboss.tools.fuse.transformation.editor.internal.l10n.Messages;
 import org.jboss.tools.fuse.transformation.editor.transformations.Function.Arg;
-
 import com.google.gson.JsonElement;
 import com.google.gson.JsonNull;
 import com.google.gson.JsonObject;
 import com.google.gson.JsonParser;
 
 public class Util {
 
-    public static final String MAIN_PATH = "src/main/"; //$NON-NLS-1$
+    private static final String MAIN_PATH = "src/main/"; //$NON-NLS-1$
 
     public static final String RESOURCES_PATH = MAIN_PATH + "resources/"; //$NON-NLS-1$
 
-    public static final String JAVA_PATH = MAIN_PATH + "java/"; //$NON-NLS-1$
+    private static final String JAVA_PATH = MAIN_PATH + "java/"; //$NON-NLS-1$
 
     public static final String[] EMPTY_STRING_ARRAY = new String[0];
 
     public static final String TRANSFORMATIONS_FOLDER = ".transformations"; //$NON-NLS-1$
 
-    public static boolean isJSONValid(String incomingJSON) {
+    public static String camelVersion(IProject project) {
```

Comment:
Sounds good. I didn't write the initial method; was just moving and renaming it when I ended up modifying it a bit internally.